src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -167,7 +167,7 @@ private static void InitSentryAndroidSdk(SentryOptions options)
             o.EnableExternalConfiguration = false;
             o.EnableDeduplication = false;
             o.AttachServerName = false;
-            o.NativeSdkName = "sentry.native.dotnet";
+            o.NativeSdkName = "sentry.native.android.dotnet";
```

Comment:
I take it this doesn't address whatever the more fundamental problem with https://github.com/getsentry/sentry-dotnet/issues/3654#issuecomment-2395681843 is?

Reply:
Yes it does.